x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/constants.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const CREATE_ENGINE_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.createEngine.title',
+  {
+    defaultMessage: 'Create an Engine',
```

Comment:
So this is something Chris has told us before when we were migrating strings, but we're actually way over-using capitalization in our current copy and it's something we need to fix before/around 7.14. He's going to eventually do a final pass, but if we can help him out with quick fixes (that he's already let us know will definitely be changed), I think that will save him & us some time down the road.

Specific to this view: `Engine` should be `engine` in almost every scenario (except where it's the first word). Source: https://github.com/elastic/kibana/pull/83504#issuecomment-729960035. Also things like `Engine Language` should just be `Engine language` (sentence casing), etc.

LMK if that's a change you're cool making as part of this PR!

Reply:
cool cool cool, I can make all those changes.  So sentence casing, we should treat all of our Nouns as nouns.